bin/init.gradle

```diff
@@ -24,6 +24,16 @@ allprojects { everyProj ->
 
                         cp.add(v.javaCompile.destinationDir.getAbsolutePath())
                     }
+
+                    def confAttrSpec = (project.hasProperty('confAttrs') 
```

Comment:
Is it possible to convert this to an `if` statement? `if (project.hasProperty('confAttrs'))`